<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="OXRight">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 212, 121);</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="OZ">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>230</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Oz:</string>
    </property>
   </widget>
   <widget class="QLabel" name="OX">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ox:</string>
    </property>
   </widget>
   <widget class="QLabel" name="RotationLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Вращать вокруг:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TranslateButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 212, 121);</string>
    </property>
    <property name="text">
     <string>Перенос</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ProjectionOYZ">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>330</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 212, 121);</string>
    </property>
    <property name="text">
     <string>Ортографическая проекция Oyz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OYLeft">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 212, 121);</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ScaleButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 212, 121);</string>
    </property>
    <property name="text">
     <string>Масштабирование</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RevertButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>410</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 212, 121);</string>
    </property>
    <property name="text">
     <string>Отменить преобразования</string>
    </property>
   </widget>
   <widget class="QLabel" name="OY">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Oy:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ProjectionOXZ">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 212, 121);</string>
    </property>
    <property name="text">
     <string>Ортографическая проекция Oxz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RevertProjection">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 212, 121);</string>
    </property>
    <property name="text">
     <string>Отменить проецирование</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OYRight">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>190</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 212, 121);</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="OXLeft">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 212, 121);</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="TransformationMatrix">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>480</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OZRight">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>250</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 212, 121);</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="OZLeft">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 212, 121);</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
    <property name="autoRepeatInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ProjectionOXY">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 212, 121);</string>
    </property>
    <property name="text">
     <string>Ортографическая проекция Oxy</string>
    </property>
   </widget>
   <widget class="QLabel" name="TransformationMatrixLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>450</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Матрица преобразования:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
